platform/android/java/nativeSrcsConfigs/CMakeLists.txt

```diff
@@ -17,4 +17,4 @@ target_include_directories(${PROJECT_NAME}
         SYSTEM PUBLIC
         ${GODOT_ROOT_DIR})
 
-add_definitions(-DUNIX_ENABLED -DVULKAN_ENABLED -DTOOLS_ENABLED)
+add_definitions(-DUNIX_ENABLED -DVULKAN_ENABLED -DANDROID_ENABLED)
```

Comment:
This is a nitpick: I fell like changes of this kind should be in their own commit since they are not really part of the main task. For me, it'd be fine the alternative of just adding a line in the commit message telling that there are some additional little fixes unrelated to the main goal.

Reply:
@RandomShaper This change is mostly cosmetic and doesn't affect the engine logic.
That CMake file is only used by Android Studio to properly show the native engine code, and so has no effects on anything else.